core/include/vcpu.h

```diff
@@ -234,6 +234,14 @@ struct vcpu_t {
     struct em_context_t emulate_ctxt;
     struct vcpu_post_mmio post_mmio;
     struct mmio_fetch_cache mmio_fetch;
+
+    // Guest CPUID feature set
+    // * The CPUID feature set is always same for each vCPU. A CPUID instruction
+    //   executed on any core will get the same result.
+    // * All vCPUs share the unique memory, which is actually allocated by the
+    //   first vCPU created by VM. If any vCPU sets features in this field, all
+    //   vCPUs will change accordingly.
```

Comment:
Why save this data in vCPU if all of them share same memory? Why not in vm?

Reply:
Yes, saving in VM has the same effect. But we considered that this IOCTL should be belonging to VCPU as it is CPUID.